core-bundle/contao/library/Contao/Combiner.php

```diff
@@ -212,9 +216,10 @@ public function getFileUrls($strUrl=null)
 
 				if ($blnDebug || !file_exists($this->strRootDir . '/' . $strPath))
```

Comment:
```suggestion
				if ($blnDebug || !file_exists($this->strRootDir . '/' . $this->strWebDir . '/' . $strPath))
```
As we dump into the web dir, the exists check probably needs to point there too?

Reply:
Or maybe change the line above from `$strPath = 'assets/'…` to `$strPath = $this->strWebDir . '/assets/'…` and use `$this->strRootDir . '/' . $strPath` in the `dumpFile()` call below too.

Reply:
> As we dump into the web dir, the exists check probably needs to point there too?

Of course 🤦‍♂️